sklearn/tests/test_isotonic.py

```diff
@@ -437,3 +438,15 @@ def test_isotonic_copy_before_fit():
     # https://github.com/scikit-learn/scikit-learn/issues/6628
     ir = IsotonicRegression()
     copy.copy(ir)
+
+
+def test_isotonic_dtype():
```

Comment:
Probably there, but not possible without estimator tags, right? ``preserves_precision``? Have you run it on ``all_estimators`` to see the status? We could post an issue with a check list.